packages/DraftEditor/src/utils/index.tsx

```diff
@@ -1,58 +1,105 @@
 import ReactDOM from "react-dom";
 import { DEFAULT_HEIGHT, LEFT, MARGIN_IMAGE, RIGHT } from "../constants";
 import eventBus from "../eventBus";
+import { MARGINS, STYLE_AUTO } from "../constants";
+
 declare let window: any;
+
 const addButtonToAlignmentToolContainer = (element: any) => {
-  if (!element) {
-    return;
-  }
-  const editor = ReactDOM.findDOMNode(element);
-  if (editor instanceof HTMLElement) {
-    const [alignmentToolContainer] = Array.from(
-      editor.querySelectorAll(".draftJsEmojiPlugin__alignmentTool__2mkQr")
-    );
+  const [alignmentToolContainer]: any = findElements(
+    element,
+    ".draftJsEmojiPlugin__alignmentTool__2mkQr"
+  );
 
-    window.showModal = () => {
-      const [focusedImage] = Array.from(
-        editor.querySelectorAll(".draftJsFocusPlugin__focused__3Mksn")
-      );
+  window.showModal = () => {
+    const [focusedImage]: any = findElements(
+      element,
+      ".draftJsFocusPlugin__focused__3Mksn"
+    );
 
-      eventBus.$emit("editImageAttributes", focusedImage.src);
-    };
+    eventBus.$emit("editImageAttributes", focusedImage.src);
+  };
 
-    alignmentToolContainer.insertAdjacentHTML(
-      "beforeend",
-      `<div class="draftJsEmojiPlugin__buttonWrapper__1Dmqh" onclick="showModal()">
+  alignmentToolContainer.insertAdjacentHTML(
+    "beforeend",
+    `<div class="draftJsEmojiPlugin__buttonWrapper__1Dmqh" onclick="showModal()">
         <i class="material-icons">
           perm_data_setting
         </i>
       </div>`
-    );
+  );
+};
+
+const findElements = (node: any, element: string) => {
+  if (!node) {
+    return [];
+  }
+  const editor = ReactDOM.findDOMNode(node);
+
+  if (editor instanceof HTMLElement) {
+    return Array.from(editor.querySelectorAll(element));
+  }
+
+  return [];
+};
+
+const updateElementStyles = (element: any, styles: any) => {
+  Object.entries(styles).forEach(([key, value]) => {
+    if (MARGINS.includes(key)) {
+      if (
+        element.style[key] !== STYLE_AUTO &&
+        element.style[key] !== STYLE_AUTO
```

Comment:
this is doing a double check on the same thing?

Reply:
to be very sure, just kidding now I'll change it

Reply:
fixed

Reply:
fixed